app/presenters/tree_builder_ops_rbac_features.rb

```diff
@@ -18,12 +18,6 @@ def initialize(name, type, sandbox, build, **params)
 
   private
 
-  def set_locals_for_render
-    locals = {:check_url => "/ops/rbac_role_field_changed/", :onclick => nil}
```

Comment:
This `onclick` was missed by https://github.com/ManageIQ/manageiq-ui-classic/pull/5350 as it is `nil` and not `false` so (obviously) not moving it to the options.